Tiltfile

```diff
@@ -55,7 +55,7 @@ k8s_yaml(
     name = 'kargo',
     namespace = 'kargo',
     values = 'hack/tilt/values.dev.yaml',
-    set = 'externalWebhooksServer.host=' + os.environ.get('KARGO_EXTERNAL_WEBHOOKS_SERVER_HOSTNAME', 'localhost:30083')
+    set = 'externalWebhooksServer.host=' + os.environ.get('KARGO_EXTERNAL_WEBHOOKS_SERVER_HOSTNAME', 'localhost')
```

Comment:
I don't think this should change. As it was originally, receiver URLs should be copy/pastable into curl to send dummy requests (from the same host) with no additional setup required.

(If that wasn't working, then I screwed something up and will happily take a look at it.)

If you want to do something better, like use Ngrok to get authentic requests from external systems through to the server, you can just set this env var in your zsh profile.